GNUmakefile

```diff
@@ -234,6 +238,14 @@ checkscripts: ## Lint shell scripts
 	@echo "==> Linting scripts..."
 	@find scripts -type f -name '*.sh' | xargs shellcheck
 
+.PHONY: checkproto
+checkproto: ## Lint protobuf files
+	@echo "==> Lint proto files..."
+	@buf check lint --config tools/buf/buf.yaml
```

Comment:
Should this be part of check as well?  Not sure why we don't run buf linters now.

Reply:
Seems fine to me!